src/contrib/persistence/Akka.Persistence.Sqlite.Tests/Bugfix4360Spec.cs

```diff
@@ -129,7 +129,7 @@ public void Should_recover_persistentactor_sqlite()
             recoveryActor.Tell("foo");
             recoveryActor.Tell("bar");
             recoveryActor.Tell(RecoverActor.DoSnapshot.Instance);
-            ExpectMsgAllOf("foo", "bar");
+            ExpectMsgAllOf(new []{ "foo", "bar" });
```

Comment:
Why is this necessary exactly? I thought the `params` handled this?

Reply:
`params` need to be replaced with `IEnumerable<T>` because we need to pass in a `CancellationToken` with a default value

Reply:
Got it, thank you. Are we going to warn users about that somehow? Roslyn rule?